cmd/subnetcmd/publish.go

```diff
@@ -395,43 +399,45 @@ func getVMInfo(sc *models.Sidecar) (*types.VM, error) {
 			return nil, errors.New("canceled by user")
 		}
 
+		url, err = app.Prompt.CaptureStringAllowEmpty("Tell us the URL to download the source. Needs to be a fixed version, not `latest`.")
```

Comment:
can we check for fixed version in capture?

Reply:
I think that's a bit overkill if we're going to have to create new Captures for every single special case. Also, we are allowing Empty here so I think it's not worth it. 

Maybe later if we're eliminating the Empty option and also have identified what the proper design for all this will be.

Reply:
https://github.com/ava-labs/avalanche-cli/issues/267